nf_requirements/v1alpha1/interface_types.go

```diff
@@ -16,18 +16,42 @@ limitations under the License.
 
 package v1alpha1
 
+import (
+	"k8s.io/apimachinery/pkg/types"
+)
+
 type AttachmentType string
 
 // AttachmentTypeNone defines an untagged attachement (no VLAN)
 const AttachmentTypeNone AttachmentType = "none"
+
 // AttachmentTypeVLAN defines a tagged/vlan attachement
 const AttachmentTypeVLAN AttachmentType = "vlan"
 
 type CNIType string
 
 // CNITypeSRIOV defines the sriov cni
 const CNITypeSRIOV CNIType = "sriov"
```

Comment:
Nitpick. Are these CNI constants defined elsewhere that we can reuse?

Reply:
I have been looking at this indeed and e.g. in NAD these are just strings. So in my research I did not find any.

Reply:
OK thanks.